apps/cave/components/UserDashboard/Summary/Staking/LiquidStakingSnapshot.tsx

```diff
@@ -0,0 +1,114 @@
+import { AirdropClaimContract } from '@concave/core'
+import { Flex } from '@concave/ui'
+import { airdropToken, getAirdropQ4ClaimableAmount } from 'components/Airdrop/Q4/airdrop'
+import { useFilterByRange } from 'components/NftFilters/Filters/hooks/useFilterByRange'
+import { useFilterByStakePool } from 'components/NftFilters/Filters/hooks/useFilterByStakePool'
+import { usePositionSorter } from 'components/NftFilters/Sorters/hooks/useNftSort'
+import { useStakePositions } from 'components/StakingPositions/DashboardBody/DashBoardState'
+import { FilterContainer } from 'components/StakingPositions/DashboardBody/FilterContainer'
+import { SnapshotLineChart } from 'components/UserDashboard/SnapshotLineChart'
+import { SnapshotTextCard } from 'components/UserDashboard/SnapshotTextCard'
+import { useStakeSettings } from 'contexts/PositionsFilterProvider'
+import { useCurrentSupportedNetworkId } from 'hooks/useCurrentSupportedNetworkId'
+import { concaveProvider } from 'lib/providers'
+import { useState } from 'react'
+import { useQuery } from 'react-query'
+import { useAccount } from 'wagmi'
+import { UserPositionCard } from '../../../StakingPositions/LockPosition/Card/UserPositionCard'
+import { DataTable } from '../../DataTable'
+import { DataTableCard } from '../../DataTableCard'
+import { SnapshotCard } from '../../SnapshotCard'
+import { SnapshotText } from '../../SnapshotText'
+import { stakechartdata } from '../dummyChartData'
+
+export const LiquidStakingSnapshot = () => {
+  const [isExpanded, setExpand] = useState(false)
+  const { address } = useAccount()
+  const stakePosition = useStakePositions()
+  const positionSorter = usePositionSorter()
+
+  const { userNonFungibleTokensInfo, totalLocked, isLoading } = stakePosition
+  const networkId = useCurrentSupportedNetworkId()
+
+  const airdropClaimableAmount = getAirdropQ4ClaimableAmount(address)
+  const airdropAmount = airdropClaimableAmount || 0
+  const { data: claimed } = useQuery(['AirdropClaimContract', networkId], async () => {
+    const airdrop = new AirdropClaimContract(concaveProvider(networkId), 'Q4')
+    return await airdrop.claimed(address)
+  })
+
+  const airdropTotal = 510691.11 //TODO
```

Comment:
@21Chani ?

Reply:
this is left over from when it was hardcoded, unsure where to get the total token distribution at the time. ideally we can pull this from somewhere reliable, maybe onchain?

Reply:
good @psgb-r3763 i did it

Reply:
new version should be good!